workspaces/web/src/components/ChallengePanel.svelte

```diff
@@ -80,11 +82,8 @@
     color: $text-invert;
     background: $yellow;
   }
-  .left {
-    float: left;
-  }
-  .right {
-    float: right;
+  .container {
```

Comment:
this is already defined by bulma https://bulma.io/documentation/layout/container/ 

perhaps you could be able to even just reuse something from bulma